sdk/keyvault/Azure.ResourceManager.KeyVault/src/autorest.md

```diff
@@ -5,17 +5,20 @@ Run `dotnet build /t:GenerateCode` to generate code.
 ``` yaml
 azure-arm: true
 library-name: KeyVault
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/954bf4ebc679ba55a6cacb39dbdacdbb956359f2/specification/keyvault/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/d29e6eb4894005c52e67cb4b5ac3faf031113e7d/specification/keyvault/resource-manager/readme.md
+tag: package-2021-10
 clear-output-folder: true
 skip-csproj: true
 namespace: Azure.ResourceManager.KeyVault
 modelerfour:
-  lenient-model-deduplication: true
+  flatten-payloads: false
 model-namespace: false
+override-operation-name:
+  Vaults_CheckNameAvailability: CheckKeyVaultNameAvailability
+  MHSMPrivateLinkResources_ListByMHSMResource: GetMhsmPrivateLinkResourcesByMhsmResource
```

Comment:
```suggestion
  MHSMPrivateLinkResources_ListByMhsmResource: GetMhsmPrivateLinkResourcesByMhsmResource
```

Reply:
And BTW I think it makes more sense if you rename this operation to `GetPrivateLinkResources` or `GetMhsmPrivateLinkResource`